mmdet/models/layers/transformer/dino_layers.py

```diff
@@ -54,9 +54,19 @@ def forward(self, query: Tensor, value: Tensor, key_padding_mask: Tensor,
                 regression results.
 
         Returns:
-            Tensor: Output queries of Transformer encoder, which is also
-            called 'encoder output embeddings' or 'memory', has shape
+            Tensor: Output queries of Transformer decoder, which is also
+            called 'decoder output embeddings' or 'memory', has shape
```

Comment:
I think `which is also called 'decoder output embeddings' or 'memory'` can be deleted. Note that `memory` is encoder outputs.